.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
its usually not advise to commit editor centric file

Reply:
Oops my bad !